pythontabcmd2/commands/extracts/encrypt_extracts_command.py

```diff
@@ -23,4 +31,10 @@ def run_command(self):
         self.encrypt_extract(server_object)
 
     def encrypt_extract(self, server):
-        pass
+        try:
+            site_id = SiteCommand.find_site_id(server, self.site_name)
+            job = server.sites.encrypt_extracts(site_id)
+            self.logger.info("Extract encrypted Successfully with "
```

Comment:
I think this command just starts encryption, we don't know if it finished/succeeded? Same for anything that returns a job id.

Reply:
Fixed it, Thank you!